### ISSUE TITLE
switch from a profile to another does not work 100% for me :(

### ISSUE THREAD

**Issue by GeraldG123** [j-a-n/lovelace-wallpanel#175]
Hi,

really great screensaver with tons of options, but I run in a problem as soon as I want to switch from one profile to another

In the raw config I entered:
```yaml
profiles:
  night:
    show_images: false
  day:
    show_images: true
profile_entity: input_text.wallpanel_profile
```
and via automation I switch from one profile to another based on time constraints
just to ensure that during the night the background is black and there will be just a clock and weather ...

but here comes the poblem:

situation 1:
– profile day is active
– wallpanel starts with profile day via automation (=some pictures)
– I end the wallpanel
– switch to profile night
– wallpanel starts with profile night via automation (=no pictures, just black background)
– I end the wallpanel
– switch back to profile day
– wallpanel starts with profile day via automation (=some pictures)
–> everything is great

situation 2:
– profile day is active
– wallpanel starts with profile day via automation (=some pictures)
– switch to profile night (and wallpanel is still active)
– wallpanel starts with profile night via automation (=no pictures, just black background)
– switch back to profile day (and wallpanel is still active)
–> still no background pictures, just black background 🙁

means move from profile night to profile day works great as long as the wallpanel is deactivated in between but not if the wallpanel is always active

any ideas?

best thanks

best regards
Gerald

**Comment by GeraldG123** [comment]
Hi,

just to clarify, the problem exists "just" for the background images,
when for example a card is profile dependent everything works fine (like with profile day show the clock card and with profile night show date card)

best regards
Gerald

**Comment by j-a-n** [comment]
Hi Gerald,

thanks for reporting!
Should be fixed with version 4.22.1.

**Comment by GeraldG123** [comment]
Hi,

thats really fantastic, everything works just great now, best thanks for the fast fix and all the best

best regards
Gerald
